pkgs/racket-doc/scribblings/reference/engine.scrbl

```diff
@@ -13,7 +13,7 @@ inspired by the work of Haynes and Friedman @cite["Haynes84"].
 Engines log their behavior via a logger with the name 
 @racket['racket/engine]. The logger is created when the module
 is instantiated and uses the result of @racket[(current-logger)]
-as its parent. The library adds logs a @racket['debug] level
+as its parent. The library adds to logs a @racket['debug] level
```

Comment:
I think just dropping `adds` is probably better: "The library logs a @racket['debug] level...".

Also, probably there should have been a hyphen between `@racket['debug]` and `level`.

Ok if I make those changes here?

Reply:
Please and thank you!